frontend/views/containers/proposals/ProposalItem.vue

```diff
@@ -148,7 +148,7 @@ export default ({
             })
           }[ruleName]()
         },
-        [PROPOSAL_GENERIC]: () => L('TODO: Change [generic] from [current] to [new-value]', {})
+        [PROPOSAL_GENERIC]: () => `${this.proposal.data.proposalData.name}`
```

Comment:
Since you're not returning a modified version of the string, there's no need to wrap this in backticks. Just return the value.